### PR TITLE
DR-516: Add link to Stackdriver logs in Gradle scan

### PR DESCRIPTION
**Pull request by fboulnois**
Add link to StackDriver logs in Gradle using timestamp and namespace.

Example Gradle scan with link: https://scans.gradle.com/s/ydlw55wzejvic

### CODE REVIEW

**Review by ddietterich — APPROVED** [review]
lgtm

File: build.gradle [hunk 1]
```diff
@@ -50,6 +50,12 @@ if (hasProperty('buildScan')) {
     buildScan {
         termsOfServiceUrl = 'https://gradle.com/terms-of-service'
         termsOfServiceAgree = 'yes'
+        def ns = System.getenv("NAMESPACEINUSE") ?: "integration-1"
```
Comment (marikomedlock) [inline build.gradle]: Perhaps change the fallback value to an invalid namespace so we're not confused (e.g. "integration-noNS")? I'm not familiar with this env var though, so I'd appreciate your thoughts.

**Review by marikomedlock — APPROVED** [review]
LGTM. One comment about the default namespace value.

File: build.gradle (hunk 1 above)
Comment (fboulnois) [reply build.gradle]: Good point. I'll talk to Jeremy about this.

**Pull request merged**